ims_python_helper/__init__.py

```diff
@@ -284,16 +363,25 @@ def get_empty_image_record_for_name(self, image_name: str, skip_existing: bool)
                 has been uploaded, and `skip_existing` is True
         """
         if skip_existing:
+            matching_uploaded_images = []
+            matching_empty_images = []
             try:
                 existing_images = self._ims_images_get()
                 for image in existing_images:
                     if image.get("name") == image_name:
                         if image.get("link"):
-                            raise ImsImageAlreadyUploaded(image)
+                            matching_uploaded_images.append(image)
                         else:
                             LOGGER.info("Image \"%s\" with ID \"%s\" found, but it has not been "
                                         "uploaded yet.", image_name, image.get("id"))
```

Comment:
This could be logged multiple times if more than one image with the same name and an empty link exists already, which would probably be pretty confusing the way it's worded right now. Perhaps, a message like this is more accurate:
```suggestion
                            LOGGER.info("Empty image \"%s\" with ID \"%s\" found."
                                        image_name, image.get("id"))
```

Or we could just log something in the `if matching_empty_images:` block below.